index.test.ts

```diff
@@ -8,188 +8,150 @@ import retry from 'async-retry';
-        await page.fill('[placeholder="New Password"]', 'password');
-        await page.fill('[placeholder="Confirm Password"]', 'password');
-        await page.click('button:has-text("Get started")');
-        await page.uncheck('input[type="checkbox"]', { force: true });
-        await Promise.all([
-          page.waitForNavigation(),
-          page.click('text=Continue'),
-        ]);
-      });
+
+    test('create user', async () => {
+      await page.goto('http://localhost:3000');
+      await page.fill('label:has-text("Name") >> .. >> input', 'Admin');
+      await page.fill(
+        'label:has-text("Email") >> .. >> input',
+        'admin@keystonejs.com'
+      );
+      await page.click('button:has-text("Set Password")');
+      await page.fill('[placeholder="New Password"]', 'password');
+      await page.fill('[placeholder="Confirm Password"]', 'password');
+      await page.click('button:has-text("Get started")');
+      await page.uncheck('input[type="checkbox"]', { force: true });
+      await page.click('text=Continue');
     });
 
-    test('change name of admin', async () => {
-      await retry(async () => {
-        await page.click('h3:has-text("Users")');
-        await Promise.all([
-          page.waitForNavigation(),
-          page.click('a:has-text("Admin")'),
-        ]);
-        await page.type('label:has-text("Name") >> .. >> input', '1');
-        await page.click('button:has-text("Save changes")');
-        await Promise.all([
-          page.waitForNavigation(),
-          page.click('nav >> text=Users'),
-        ]);
-        expect(await page.textContent('a:has-text("Admin1")')).toBe('Admin1');
-      });
+    test('change admin name', async () => {
+      await page.click('h3:has-text("Users")');
+      await page.click('a:has-text("Admin")');
+      await page.type('label:has-text("Name") >> .. >> input', '1');
+      await page.click('button:has-text("Save changes")');
+      await page.click('nav >> text=Users');
+      expect(await page.textContent('a:has-text("Admin1")')).toBe('Admin1');
     });
 
-    test('create post', async () => {
-      await retry(async () => {
-        await Promise.all([
-          page.waitForNavigation(),
-          page.click('nav >> text=Posts'),
-        ]);
-        await page.click('a:has-text("Create Post")');
-        await page.fill('input[type="text"]', 'content');
-        await Promise.all([
-          page.waitForNavigation(),
-          page.click('button:has-text("Create Post")'),
-        ]);
-        await page.type('input[type="text"]', '1');
-        await page.click('button:has-text("Save changes")');
-      });
+    test('create a post', async () => {
+      await page.click('nav >> text=Posts');
+      await page.click('a:has-text("Create Post")');
+      await page.fill('input[type="text"]', 'title');
+      await page.click('button:has-text("Create Post")');
+      await page.waitForTimeout(2000); // sleep for a second or two
```

Comment:
The problem is `Save changes` isn't actually ready, could probably fix that later - this is fine for now